debian/changelog

```diff
@@ -1,3 +1,10 @@
+opx-base-model (3.146.0+opx1) unstable; urgency=medium
+
+  * Update: Changing the cps dependancy lower limit in order to omit CPS
```

Comment:
Can you please update the comments ...e.g. "Fix the dependency to get the CPS fix for openapi omit".